docs/source/yelpsoa_configs.rst

```diff
@@ -169,6 +169,16 @@ These options are only applicable to tasks scheduled on Kubernetes.
     documentation on `topology spread constraints
     <https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/>`_.
 
+  * ``unhealthy_pod_eviction_policy``: defines when unhealthy pods can be evicted under a PodDisruptionBudget
+
+      unhealthy_pod_eviction_policy: IfhealthyBudget
```

Comment:
or, if we don't like the suggestion below, we should fix the casing typo 
```suggestion
      unhealthy_pod_eviction_policy: IfHealthyBudget
```

and maybe state that this is the default more explicitly?